mautic/mautic
# Mail status count display

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | No
| New feature? | No (Minor enhancement)
| Related user documentation PR URL | https://mautic.org/docs/en/campaigns/index.html
| Issues addressed (#s or URLs) | No
| BC breaks? | No
| Deprecations? | No

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
Showing mail send count along with the percentage value

#### Steps to test this PR:
1. After sending campaign mails, check-on the Campaign view page
2. Please check the "Action" tab, there we can see the actual count of mail send
![55 hubs by 55 weeks](https://user-images.githubusercontent.com/10077578/27168179-3417810e-51c2-11e7-9ab8-14f090557518.png)
